gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/purger/HivePurgerSource.java

```diff
@@ -242,21 +200,26 @@ protected void createWorkunitsFromPreviousState(SourceState state) {
         continue;
       }
       WorkUnit workUnit = workUnitState.getWorkunit();
-      Preconditions.checkArgument(workUnit.contains(HivePurgerConfigurationKeys.PARTITION_NAME),
+      Preconditions.checkArgument(workUnit.contains(ComplianceConfigurationKeys.PARTITION_NAME),
           "Older WorkUnit doesn't contain property partition name.");
-      int executionAttempts = workUnit.getPropAsInt(HivePurgerConfigurationKeys.EXECUTION_ATTEMPTS,
-          HivePurgerConfigurationKeys.DEFAULT_EXECUTION_ATTEMPTS);
+      int executionAttempts = workUnit.getPropAsInt(ComplianceConfigurationKeys.EXECUTION_ATTEMPTS,
+          ComplianceConfigurationKeys.DEFAULT_EXECUTION_ATTEMPTS);
       if (executionAttempts < this.maxWorkUnitExecutionAttempts) {
```

Comment:
Those WorkUnits will be tried again in the next cycle. We will add alerting later.